bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsComparator.java

```diff
@@ -34,7 +34,7 @@ public int compare(Obs o1, Obs o2) {
 			return direction * ((ret != 0) ? ret : -o1.compareTo(o2));
 		}
 		
-		return direction * attr.getComparator().compare(o1, o2);
+		return direction * attr.getComparator(variant.getSchedule().getMiddlePoint()).compare(o1, o2);
```

Comment:
I'm retiring what I proposed above. I can't find a *better* date where `CandidateObsComparator` is [instantiated](https://github.com/gemini-hlsw/ocs/blob/4fd9f5910d33c82a9793232476503edd63dc46b9/bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsViewAdvisor.java#L70).